AndroidSDK/src/com/leanplum/LeanplumNotificationChannels.java

```diff
@@ -0,0 +1,578 @@
+package com.leanplum;
```

Comment:
This class is way too big, maybe split into channels and channel groups. And since all the methods are static, can call them `LeanplumNotificationChannelUtil`, `LeanplumNotificationChannelGroupUtil`.